src/components/FileTree/index.tsx

```diff
@@ -135,29 +137,57 @@ export type PublicProps = {
   version: Version;
 };
 
-type Props = PublicProps;
+type Props = PublicProps & ConnectedReduxProps;
 
 export class FileTreeBase extends React.Component<Props> {
+  tree: DirectoryNode;
+
+  constructor(props: Props) {
+    super(props);
+    const { version } = this.props;
+    this.tree = buildFileTree(
```

Comment:
This optimization makes sense (good idea) but I'm not seeing much of a speedup after this change. Also, the tree should be updated when the `version` prop changes. I don't think it's worth finishing out this change for this patch. We should address the performance issues in a separate issue.

Reply:
Ok, I will revert the commit that introduced that change.